test_istgt.sh

```diff
@@ -2006,6 +2006,20 @@ run_io_timeout_test()
 		# Test to verify disconnection of replica if delay from replica is more than maxiowait
 		$ISTGTCONTROL maxiowait 5
 		$IOPING  -c 3 -B -WWW /dev/$device_name > $iopinglog
+		cmd="grep -ch 'changing IO max wait time from 60 to 120' $LOGFILE"
+		val=$(eval $cmd)
+		if [ $val -ne 1 ]
```

Comment:
count won't match because we are not cleaning the `/tmp/istgt.log`. Better to update `stop_istgt` with `rm $LOGFILE`.

Reply:
fixed it by checking for 'eq 0' for error